superset-frontend/src/features/allEntities/AllEntitiesTable.tsx

```diff
@@ -30,6 +30,7 @@ import { EmptyStateBig } from 'src/components/EmptyState';
 import { fetchObjects } from '../tags/tags';
 
 const MAX_TAGS_TO_SHOW = 3;
+const PAGE_SIZE = 5;
```

Comment:
Should this be 50, like before?

Reply:
design wants this to be 10 since there are 3 tables on the 1 page